components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html

```diff
@@ -205,7 +205,7 @@
             {% endif %}
 
             {% if not share %}
-            {% if image and not omeTiffDisabled %}
+            {% if image %}
```

Comment:
This commit is not related to the description of the PR
Why did you include it?

Reply:
Could you put the commit in another PR since the change is actually not tested the same way